### DISCUSSION TITLE
Translucent gray cover over GitHub Docs page

### DISCUSSION THREAD

**Discussion by audamoon** [community/community#28768] [Other Feature Feedback, Questions, & Ideas] [answered]
On all my browsers (Edge, Chrome, FireFox beta on mobile) I have this strange gray cover over all GitHub Docs page. It disappears after a few minutes, but it's kind of annoying. I tried to disable adblock, but it doesn't help
![Снимок экрана 2022-08-08 171643](https://user-images.githubusercontent.com/77841503/183439984-b927b83a-6b8f-43cd-8258-60cf720c8be1.png)

**Comment by airtower-luna** [comment]
I see it too (Firefox 103 on Linux), very annoying.

**Comment by melvincwng** [comment, marked as answer]
@audamoon Hi Nikita, thanks for highlighting this issue to GitHub Community.

On my end, and as above mentioned by other users, we are all seeing this translucent gray cover on the GitHub Docs pages.

My suggestion would be for you to raise a ticket to [GitHub Bug Report](https://support.github.com/contact/bug-report) over here, so that the engineers at GitHub are directly aware of this issue.

Hope this helps.
